lib/constructor/index.js

```diff
@@ -72,8 +72,8 @@ function Constructor(app, options) {
   this.purgeRetries = (options.retries || 0) + 5;
 
   this.source = options.source;
-  this.target = options.target;
-  this.rootDir = options.target || os.tmpdir();
+  this.target = options.target || os.tmpdir();
+  this.rootDir = options.target || this.target;
```

Comment:
Couldn't this just be `this.roorDir = this.target`?

Reply:
probably, i doubt we want to support getters on `options`